pom.xml

```diff
@@ -121,13 +121,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.10</version>
+            <version>2.26-beta-1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-durable-task-step</artifactId>
-            <version>2.3</version>
+            <version>2.22-rc713.c51e25cf0eef</version> <!-- TODO https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/75 -->
```

Comment:
https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/75, to get `ABORTED` rather than `FAILURE` in the test.

Note that this build is actually _older_ than the 2.22 on the update center, but it does contain https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/63 and the subsequent commits are uninteresting.